tests/spec/core/examples-spec.js

```diff
@@ -145,7 +145,7 @@ describe("Core — Examples", () => {
     const ops = makeStandardOps({}, body);
     const doc = await makeRSDoc(ops);
     const mybutton = doc.getElementById("mybutton");
-    expect(mybutton.onclick).toBeDefined();
```

Comment:
This didn't really work as `toBeDefined()` is effectively `not.toBeUndefined()` 😅